backend/internal/service/handler/users/review.go

```diff
@@ -0,0 +1,44 @@
+package users
+
+import (
+	"platnm/internal/storage"
+
+	"github.com/gofiber/fiber/v2"
+
+	"platnm/internal/errs"
+)
+
+type ReviewHandler struct {
+	reviewRepository storage.ReviewRepository
+	userRepository storage.UserRepository
+}
+
+func NewReviewHandler(reviewRepository storage.ReviewRepository, userRepository storage.UserRepository) *ReviewHandler {
+	return &ReviewHandler{
+		reviewRepository,
+		userRepository,
+	}
+}
+
+func (h *ReviewHandler) GetReviewsByUserID(c *fiber.Ctx) error {
+
+	id := c.Params("id")
+
+	exists, err := h.reviewRepository.UserExists(c.Context(), id)
```

Comment:
see prev comments about moving UserExists to UserRepository, and then you can update it here too so that you're using the additional repository in this handler
```suggestion
	exists, err := h.userRepository.UserExists(c.Context(), id)
```

Reply:
Fixed